packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -114,8 +114,7 @@ export class ExpressionParser {
    */
   setScope(scope = {}) {
     Object.entries(scope).forEach(([name, value]) => {
-      const expressionValue = value || 0;
-      this.set(name, expressionValue);
+      this.set(name, value);
```

Comment:
Great change, very much approve! The defaulting behaviour was quite unexpected, and could break lots of behaviour.

Just a note that I renamed this function in https://github.com/beyondessential/tupaia/pull/3173/files#diff-6d258c85fc6c829d3bcd8670f8d94362dba252a275bbe68374799674c494c03e but I don't think changes should conflict...